scripts/exgdas_enkf_update.sh

```diff
@@ -204,7 +204,7 @@ for imem in $(seq 1 $NMEM_ENS); do
    for FHR in $nfhrs; do
       ${NLN} "${COMIN_ATMOS_HISTORY_MEM_PREV}/${GPREFIX}atmf00${FHR}${ENKF_SUFFIX}.nc" \
          "sfg_${PDY}${cyc}_fhr0${FHR}_${memchar}"
-      if [[ "${DO_GSISOILDA}" = "YES" ]]; then
+      if [[ "${hofx_2m_sfcfile}" = ".true." ]]; then
```

Comment:
Where does the value of `hofx_2m_sfcfile` get set?

Reply:
It's set in [config.anal](https://github.com/NOAA-EMC/global-workflow/blob/16219a3e6904662b774b3b23edc576b885737b3b/parm/config/gfs/config.anal#L52)

I made this change so that we can monitor the 2m obs without assimilating them.

Reply:
Is it defined at the top of this script in some style like:
`hofx_2m_sfcfile=${hofx_2m_sfcfile:-".false."}`
if not, we should probably do that

Reply:
Thanks for pointing me to where it's set @ClaraDraper-NOAA . I also second @CoryMartin-NOAA's suggestion.

Reply:
done.